pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -85,6 +95,20 @@ const (
 	CleanPodPolicyNone      CleanPodPolicy = "None"
 )
 
+// CleanPolicy defines the policy type of cleaning up tf-jobs.
```

Comment:
should we group CleanPolicy and CleanDelay as a single object as such ?
CleanPolicy {
      CleanPolicyType (AllPods, RunningPods, Job, None)
      CleanDelay
}